doc/rst/usingchapel/QUICKSTART.rst

```diff
@@ -21,7 +21,7 @@ full-featured version of Chapel, refer to
    packages you should have available to build and run Chapel.
 
 
-1) If you don't already have the Chapel 1.25.1 source release, see
+1) If you don't already have the Chapel 1.25.0 source release, see
```

Comment:
Sorry, I wasn't clear.  This should either be simply '1.25' (i.e., be agnostic to the update version) or else be '1.25.1' (i.e., be more specific to this specific release than we typically are).  Either is fine with me.